### ISSUE TITLE
MACD signal not working

### ISSUE THREAD

**Issue by sahilmen** [mrjbq7/ta-lib#468]

**Comment by trufanov-nok** [comment]
Works for me:
```
import talib as ta
import numpy as np

closes = [0.843707911389933, 0.765970112007782, 0.348204974142001, 0.770534027420264, 0.411114771704617, 0.520720835914186, 0.505732138772406, 0.126974758689069, 0.793145540635449, 0.555526153111841, 0.715492909980816, 0.10350937402732, 0.518999541773795, 0.066738752009136, 0.299164079993025, 0.058279108076808, 0.285816961215701, 0.52674626788859, 0.691841436250272, 0.60346943930196, 0.668561932588703, 0.32799385807473, 0.062774772706994, 0.124470451309067, 0.961870073163633, 0.317952952456611, 0.552529191593644, 0.919507369595207, 0.075544634422029, 0.062974891504709, 0.195180501312462, 0.637447185925008, 0.344309751457209, 0.334677348920507, 0.207965145060056, 0.17003964029082, 0.994304899295053, 0.683478619553302, 0.325747400579953, 0.141959011664527, 0.34349303395525, 0.533535308589003, 0.228959380725431, 0.300826531717415, 0.167710704628889, 0.786384987626858, 0.053212991068855, 0.029144101167118, 0.515780108167168, 0.713826724404015, 0.736800863061552]

macd1, signal1, hist1 = ta.MACD(np.array(closes), 12, 26, 9)
macd2, signal2, hist2 = ta.MACD(np.array(closes), 12, 26, 3)

hist1
hist2
```

```
  >>> hist1
array([        nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan, -0.00305579, -0.0109813 ,
       -0.01741101,  0.03249657,  0.04258608,  0.02409905, -0.00028438,
       -0.00249431,  0.00858183, -0.00418014, -0.00718113, -0.01697198,
        0.01751085, -0.00837568, -0.02547113, -0.00351029,  0.02336995,
        0.04069849])
>>> hist2
array([        nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,  0.03261805, -0.00116277, -0.01460988,
       -0.01270131,  0.00744464,  0.00306426,  0.0008656 , -0.00490262,
       -0.00784254,  0.02509459,  0.02291531,  0.00522652, -0.009614  ,
       -0.00622376,  0.00349892, -0.00515404, -0.00497516, -0.0095045 ,
        0.01467802, -0.00665121, -0.01505722,  0.00301302,  0.01786787,
        0.02268552])
```

**Comment by trufanov-nok** [comment]
Works for me:
```
import talib as ta
import numpy as np

closes = [0.843707911389933, 0.765970112007782, 0.348204974142001, 0.770534027420264, 0.411114771704617, 0.520720835914186, 0.505732138772406, 0.126974758689069, 0.793145540635449, 0.555526153111841, 0.715492909980816, 0.10350937402732, 0.518999541773795, 0.066738752009136, 0.299164079993025, 0.058279108076808, 0.285816961215701, 0.52674626788859, 0.691841436250272, 0.60346943930196, 0.668561932588703, 0.32799385807473, 0.062774772706994, 0.124470451309067, 0.961870073163633, 0.317952952456611, 0.552529191593644, 0.919507369595207, 0.075544634422029, 0.062974891504709, 0.195180501312462, 0.637447185925008, 0.344309751457209, 0.334677348920507, 0.207965145060056, 0.17003964029082, 0.994304899295053, 0.683478619553302, 0.325747400579953, 0.141959011664527, 0.34349303395525, 0.533535308589003, 0.228959380725431, 0.300826531717415, 0.167710704628889, 0.786384987626858, 0.053212991068855, 0.029144101167118, 0.515780108167168, 0.713826724404015, 0.736800863061552]

macd1, signal1, hist1 = ta.MACD(np.array(closes), 12, 26, 9)
macd2, signal2, hist2 = ta.MACD(np.array(closes), 12, 26, 3)

signal1
signal2
```
```
>>> signal1
array([        nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan, -0.03921163, -0.04195696,
       -0.04630971, -0.03818557, -0.02753905, -0.02151429, -0.02158538,
       -0.02220896, -0.0200635 , -0.02110854, -0.02290382, -0.02714681,
       -0.0227691 , -0.02486302, -0.0312308 , -0.03210838, -0.02626589,
       -0.01609127])
>>> signal2
array([        nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan,         nan,         nan,         nan,
               nan,         nan, -0.02603356, -0.02719633, -0.04180621,
       -0.05450752, -0.04706287, -0.04399862, -0.04313302, -0.04803564,
       -0.05587818, -0.03078359, -0.00786828, -0.00264176, -0.01225576,
       -0.01847951, -0.01498059, -0.02013463, -0.02510979, -0.03461429,
       -0.01993627, -0.02658749, -0.04164471, -0.03863169, -0.02076381,
        0.00192171])
```
